### ISSUE TITLE
Class 'PhpOffice\PhpWord\Reader\Style\Font' not found

### ISSUE THREAD

**Issue by a761208** [PHPOffice/PHPWord#1101]
/vendor/phpoffice/phpword/src/PhpWord/Reader/MsDoc.php at line 1693

**Comment by a761208** [comment]
Below is my code:
```php
$file = 'path/to/vendor/phpoffice/phpword/samples/resources/Sample_11_ReadWord97.doc';
$phpWord = IOFactory::load($file, 'MsDoc');
exit();
```

**Comment by a761208** [comment]
There is no use section for `Style\Font`:
```php
1692:case 0x07:
1693:    $oStylePrl->styleFont['underline'] = Style\Font::UNDERLINE_DASH;
1694:    break;
```

**Comment by troosan** [comment]
This is fixed (#892) in the latest development branch (and will be part of the 0.14 release)
